README.md

```diff
@@ -129,6 +129,15 @@ Job.objects.create(name='critical_job', priority=2)
 
 Jobs will be ordered by their `priority` (highest to lowest) and then the time which they were created (oldest to newest) and processed in that order.
 
+### Scheduling jobs
+If you'd like to create a job but have it run at some time in the future, you can use the `run_after` field on the Job model:
+
+```python
+Job.objects.create(name='scheduled_job', run_after=timezone.now() + timedelta(minutes=10))
+```
+
+Of course, the job will only be run if your `python manage.py worker` process is running at the time when the job is scheduled to run. Otherwise, it will run the next time you start your worker process after that time has passed.
```

Comment:
is it worth noting that this isn't exact? say if you're queue has a large backlog it may be sometime before the worker can pick it up

Reply:
Oh that's a good point.

Reply:
Have added a note on that. A large backlog _shouldn't_ be so much of an issue, because jobs are picked up in order of creation - so scheduled jobs were probably created a long time ago and so will be picked up before any other jobs that have been created more recently. However if a job is currently being processed then of course your scheduled job will only run after it's finished.